### ISSUE TITLE
npm flowbite-vue not found

### ISSUE THREAD

**Issue by blackavec** [themesberg/flowbite-vue#22]
not sure if this is a abandoned project or something else is wrong but i cannot install flowbite-vue.

even link https://www.npmjs.com/package/flowbite-vue is showing 404

**Comment by phenomenal-hardy** [comment]
having exact same problem :(, any tips to get it working?

**Comment by Alex-Golovin** [comment]
waiting for... one of team member in Discord flowbite community said that he about to publish package until the end of the week

**Comment by pgrazek** [comment]
Is it published?

**Comment by Chank1e** [comment]
@pgrazek @Alex-Golovin @phenomenal-hardy @humyongwon @blackavec just released v 0.0.1, now you can use it)
[https://www.npmjs.com/package/flowbite-vue](https://www.npmjs.com/package/flowbite-vue)
Now only this components are implemented
```typescript
import{ Spinner, Button, ButtonGroup, Alert, Tabs, Tab, Dropdown, FlowbiteThemable, ListGroup, ListGroupItem } from 'flowbite-vue'
```
